pybind11/how-to.rst

```diff
@@ -468,14 +525,15 @@ Edit ``python/.../SConscript`` to look like this:
     from lsst.sconsUtils import scripts
     scripts.BasicSConscript.pybind11(
         [
-            'exampleOne',
+            '_tmpl',
             # ... list additional pybind11 wrappers, if any
         ],
         addUnderscore=False,
     )
 
 ``addUnderscore`` is historical baggage; eventually we plan to make ``False`` the default.
 
+
```

Comment:
I guess updating sconsUtils at this time isn't worth it?

Reply:
I'm not sure we've stopped using it everywhere.  These changes also make using the current signature of this sconsUtils  signature awkward in other ways - now there's usually only a single module, and that means we use the `extraSrc` option a lot more.  It probably makes the most sense to fix both of those at the same time once we're sure the old approaches are no longer being used (leaving aside the question of when that might be).